src/components/Form.js

```diff
@@ -0,0 +1,183 @@
+import React from "react";
+import { useForm } from "react-hook-form";
+import InputText from "./InputText";
+import InputNumber from "./InputNumber";
+import InputRadio from "./InputRadio";
+import InputTextArea from "./InputTextArea";
+import InputCheckBox from "./InputCheckBox";
+
+function Form() {
+  const { handleSubmit, errors, register, watch } = useForm({
+    mode: "onChange",
+  });
+
+  function onFormSubmit(data) {
+    console.log(data);
+    const html = `
+    <h2>Gegevens</h2>
+    <p>Naam: ${data.firstName} ${data.lastName}</p>
+    <p>Leeftijd: ${data.age}</p>
+    <p>Postcode: ${data.postalCode}</p>
+    <p>Huisnummer: ${data.houseNumber}</p>
+    <p>Bezorgfrequentie: ${
+      data.differentValue ? data.differentValue : data.delivery
+    }</p>
+    <p>Opmerkingen: ${data.comments}</p>
+    `;
+    const orderPlaced = document.querySelector("#order-placed");
+    orderPlaced.innerHTML = "";
+    orderPlaced.insertAdjacentHTML("afterbegin", html);
```

Comment:
Ik vind die niet echt mooi. Als je geen React had gehad had ik het gesnapt, maar nu kun je juist de kracht van React gebruiken zodat dit niet meer hoeft. Je had deze p-tags al klaar kunnen zetten met alle data erin en misschien met een boolean kunnen checken of het ge-submit is of niet